virt/about-virt.adoc

```diff
@@ -20,4 +20,23 @@ You can check your {VirtProductName} cluster for compliance issues by installing
 :FeatureName: {VirtProductName} integration with the Compliance Operator
 include::snippets/technology-preview.adoc[]
 
-include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+
+[id="virt-snocaps-differences_{context}"]
+= Single Node {product-title} differences
+
+You can install {VirtProductName} on a single-node cluster. See xref:../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[About {sno}] to prepare your cluster for installation.
+
+When provisioning an {product-title} {sno} cluster with the link:https://cloud.redhat.com/blog/using-the-openshift-assisted-installer-service-to-deploy-an-openshift-cluster-on-metal-and-vsphere[assisted installer], out-of-the-box persistent storage is auto-deployed.
```

Comment:
Agree on #1 and #3.

I know that "out-of-the-box" is a bit market-y, but I don't think that "default" really conveys the premade readiness of the solution. Maybe "pre-configured" would work? WDYT? I can ask the SME if that is accurate.

Reply:
Okay so I just checked and the ISG does say to avoid out-of-the-box. So yeah I'd recommend to check with your SME to find an alternative term that conveys the same meaning. Thanks!

Reply:
Cool, thanks.